apm-sniffer/config/logconfig.yaml

```diff
@@ -0,0 +1,12 @@
+- name: "log4j"
```

Comment:
license header

Reply:
Put this file into a sub-directory (e.g. `/config/logger-plugin/logconfig.yaml`)?